src/components/QuranReader/index.tsx

```diff
@@ -115,10 +117,26 @@ const QuranReader = ({
     }
   };
 
+  const onCopy = () => {
+    const selection = window.getSelection();
+    const QuranWordsToCopy = Array.from(document.querySelectorAll(`[data-word-position]`))
```

Comment:
Would it be better to introduce a constant with the value of `data-word-position` that can be used here and also inside `QuranWord/QuranWord.tsx`.